airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -91,7 +91,11 @@ def _get_org_repositories(config: Mapping[str, Any], authenticator: MultipleToke
                 unchecked_repos.add(org_repos)
 
         if unchecked_orgs:
-            stream = Repositories(authenticator=authenticator, organizations=unchecked_orgs)
+            stream = Repositories(
+                authenticator=authenticator, 
+                organizations=unchecked_orgs,
+                api_url=config["api_url"]
```

Comment:
I'd recommend to do:
```suggestion
                api_url=config.get("api_url", GITHUB_API_DEFAULT)
```
This way you're not going yo introduce a breaking change to the connector.

Reply:
> I'd recommend to do:
> 
> This way you're not going yo introduce a breaking change to the connector.

@marcosmarxm Thanks for the review! I added the requested change in both places the call is made in [6ba82cc4e27c66416d0bb43488063f8fc62a62e3](https://github.com/airbytehq/airbyte/pull/30076/commits/6ba82cc4e27c66416d0bb43488063f8fc62a62e3). Ran unit_tests, check, discover and read after making the update. Still good.